ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -0,0 +1,376 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import utils 1.0
+
+import SortFilterProxyModel 0.2
+
+Control {
+    id: root
+
+    property alias currentTabIndex: stackLayout.currentIndex
+    property bool isCurrentUser
+    property bool readOnly
+    property var profileStore
+    property var walletStore
+    property var communitiesModel
+
+    signal closeRequested()
+
+    horizontalPadding: readOnly ? 20 : 40 // smaller in settings/preview
```

Comment:
Might be better to add the paddings in the GridViews or detach the scrollbar. Having it here will squeeze the entire grid view and the scrollbar is touching the cells even tho there is lots of space to the right.

https://user-images.githubusercontent.com/47811206/223198794-96ef0e7b-edc9-46b6-ae93-d6c27af053a0.mov

Reply:
Yup, good idea

Reply:
Fixed